{"issue_type": "request_error", "error_msg": "HTTPConnectionPool(host='www.co.linn.or.us', port=80): Max retries exceeded with url: /robots.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f86e571e490>, 'Connection to www.co.linn.or.us timed out. (connect timeout=15)'))"}